modules/ROOT/pages/dataweave-type-reuse-json-schema.adoc

```diff
@@ -0,0 +1,262 @@
+= Reusing Types from JSON Schema
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+JSON schema is a standard providing a format for what JSON data is required for a given application and how to interact with it. Many existing data formats or types are expressed using this standard. In particular, Rest APIs described by a RAML file are most likely to have their types defined in JSON schemas.
+
+In order to reuse these already defined types, you can load the JSON schema into DataWeave leaving the types structure ready to be used in your scripts.
+
+The JSON schema loader parses schemas following the https://json-schema.org/specification-links.html#draft-7[JSON Schema Draft 7].
+
+== Import a Type from a JSON Schema
+
+Having a JSON schema in the resources: 
```

Comment:
What do we mean by "Having a JSON schema in the resources:" here?

Reply:
I meant that the below file example (Person.json) is in the resources directory.